RockLib.HealthChecks.AspNetCore/Collector/CollectorOptions.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace RockLib.HealthChecks.AspNetCore.Collector;
+
+/// <summary>
+/// Options for configuring a <see cref="HealthMetricCollector"/>.
+/// </summary>
+/// <remarks>Intended to be reused in other IMetricCollector implementations.</remarks>
+public record CollectorOptions
```

Comment:
I'm unsure if we'd want this to be named something more specific.  Anyone else have thoughts?

Reply:
The 'Collector', as an entire package, is independent of the health check and designed for reuse.  In a later phase the collector will collect database observations.  It could collect anything - e.g. Redis and/or Kafka interactions.  For those reasons I chose Collector.